crates/ra_hir_def/src/item_scope.rs

```diff
@@ -136,6 +140,10 @@ impl ItemScope {
         self.legacy_macros.insert(name, mac);
     }
 
+    pub(crate) fn define_unnamed_trait(&mut self, def: TraitId, vis: Visibility) {
+        self.unnamed_traits.insert(def, vis);
```

Comment:
there's an edge case here if we import a trait multiple times with different visibility; I guess we should actually take the 'largest' visibility. E.g. if we do
```rust
pub use prelude_a::*;
use prelude_b::*;
```
we'd expect a trait that was exported without name by both to be exported again.